tactics/cbn.ml

```diff
@@ -38,34 +40,34 @@ module Cst_stack = struct
 - constant applied to params = term in head applied to args
 - there is at most one arguments with an empty list of args, it must be the first.
 - in args, the int represents the indice of the first arg to consider *)
-  type t = (constr * constr list * (int * constr array) list)  list
+  type t = (constr * constr list * (int * constr array) list * volatile)  list
```

Comment:
I know it requires a larger diff, but would it be possible to get a record type here? This is starting to be really to big for its own good.

Reply:
The pattern matchings get pretty painful
I can try again if you insist

Reply:
Nah, I've tried that at some point as well and let the patch die. It was indeed annoying. Better merge this as-is then.